### PR TITLE
Use ${group} in quilt.mod.json

### PR DESCRIPTION
**Pull request by TheKodeToad**
I don't know how important the thing I removed was (like with most PRs /hj). I thought about also expanding the ID, but I don't know if it would cause issues if it had spaces. lmk your thoughts, though!

Hopefully I can make less boring contributions to the project in future! :P

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,10 +36,8 @@ dependencies {
 }
 
 processResources {
-	inputs.property 'version', version
-
```
Comment (OroArmor) [inline build.gradle]: These lines are needed as the task wont be updated if the version changes.

Comment (TheKodeToad) [reply build.gradle]: Ahh... makes sense

**Comment by TheKodeToad** [comment]
Ahh, it seems to cause issues when running in Eclipse. But the same applies for the version field.

**Review by OroArmor — APPROVED** [review]

**Comment by TheKodeToad** [comment]
I feel like it's important for me to say that before this commit it launches fine in Eclipse; but it won't after. It's fairly trivial to fix though (just manually add the group or run the Gradle task).

**Review by OroArmor — APPROVED** [review]

**Comment by eristhea** [comment]
conflicts resolved and changes merged by #33. thanks!

**Pull request closed without merging**